### PR TITLE
Some Minor Fixes and Improvements

### PR DESCRIPTION
**Pull request by omfgitsmark**
Mostly small formatting and quality of life changes. Here are my changelog notes:

7-10-29
- Added "prices" table.
- Tidy'd up: fixed some minor formatting, and made some small improvements to log output.
- Added option of using "AllSetFiles" for input, simply provide a path to the directory as the input.
- Added auto-increment "id" column as primary key to relational tables for easy referencing.

### CODE REVIEW

File: mtgsqlive/json2sql.py [hunk 1]
```diff
@@ -46,9 +45,14 @@ def validate_io_streams(input_file: pathlib.Path, output_file: pathlib.Path) ->
     :param output_file: Output file (SQLite)
     :return: Good to continue status
     """
-    if not input_file.is_file():
-        LOGGER.fatal("Input file {} does not exist.".format(input_file))
-        return False
+    if input_file.is_file():
+        LOGGER.info("Building using AllSets.json master file.")
+    else:
```
Comment (ZeldaZach) [inline mtgsqlive/json2sql.py:53]: if/elif/else instead of nested if/else

File: mtgsqlive/json2sql.py [hunk 2]
```diff
@@ -347,13 +398,16 @@ def handle_legal_rows(
     """
     legalities = []
     for card_format, format_status in card_data["legalities"].items():
-        legalities.append(
-            {"uuid": card_uuid, "format": card_format, "status": format_status}
-        )
+        # I use an assumed "legal" system to save some space/queries, only insert rows with non-legal status
+        # If a query returns no results you can assume its legal in the format queried.
+        # "Innocent Until Proven Guilty" :D
```
Comment (ZeldaZach) [inline mtgsqlive/json2sql.py]: _ehhhhhh_ not sure if I agree with this

File: mtgsqlive/json2sql.py [hunk 3]
```diff
@@ -467,7 +544,7 @@ def modify_for_sql_insert(data: Any) -> Union[str, int, float]:
         return ", ".join(data)
 
     if isinstance(data, bool):
-        return int(data == "True")
+        return int(data == True)
```
Comment (ZeldaZach) [inline mtgsqlive/json2sql.py]:
```suggestion
        return int(data)
```

**Review by ZeldaZach — CHANGES_REQUESTED** [review]
JSON2sql on the outside should be moved inside. Maybe add a flag via argparse to the python program so it will output SQL or SQLite (default to lite)?

Some other minor comments

File: mtgsqlive/json2sql.py (hunk 1 above)
Comment (omfgitsmark) [reply mtgsqlive/json2sql.py:53]: ha yeah, I used elif later in the parse_and_import_cards function so I definitely know to use it, but my brain doesn't always do what I want. Thanks for catching it.

File: mtgsqlive/json2sql.py (hunk 2 above)
Comment (omfgitsmark) [reply mtgsqlive/json2sql.py]: That's why I commented this to explain it, it's really a matter of opinion. I'll leave it up to you.

Comment (ZeldaZach) [reply mtgsqlive/json2sql.py]: Yeah, I'm gonna want all entries. Some data can be inferred, this isn't easy to infer.

**Comment by omfgitsmark** [comment]
All suggested changes made, and cleaned up.

EDIT: Also I think what you were suggesting in your original reply was to output to other forms of SQL, and I plan on at least adding MySQL when I get some free time, then go from there.

**Review by ZeldaZach — APPROVED** [review]
Good enough for me, nice change over!

**Pull request merged**